ports/stm32/mpconfigboard_common.h

```diff
@@ -181,11 +181,26 @@
 #error Unsupported MCU series
 #endif
 
-// Configure HSE for bypass or oscillator
-#if MICROPY_HW_CLK_USE_BYPASS
-#define MICROPY_HW_CLK_HSE_STATE (RCC_HSE_BYPASS)
-#else
-#define MICROPY_HW_CLK_HSE_STATE (RCC_HSE_ON)
+// Configure clock source for the chip
+#if MICROPY_HW_CLK_SRC_HSI
```

Comment:
To be consistent with other config names, probably better to call this `MICROPY_HW_CLK_USE_HSI` instead (see eg `MICROPY_HW_RTC_USE_LSE`).